frontend/src/components/Header.style.ts

```diff
@@ -9,13 +9,16 @@ export const FmuLogo = styled.img`
 `;
 
 export const ProjectInfoContainer = styled.div`
-  padding: 0.5em;
+  padding: 0.4em;
+  height: ${tokens.spacings.comfortable.x_large};  
   border: solid 1px ${tokens.colors.ui.background__medium.hex};
   background: ${tokens.colors.ui.background__light.hex};
-  color: ${tokens.colors.text.static_icons__secondary.hex};
   border-radius: ${tokens.shape.corners.borderRadius};
+  display: flex;
+  align-items: center;
```

Comment:
Suggest ordering these as follows:

```suggestion
  height: ${tokens.spacings.comfortable.x_large};
  padding: 0.4em;
  border: solid 1px ${tokens.colors.ui.background__medium.hex};
  border-radius: ${tokens.shape.corners.borderRadius};
  background: ${tokens.colors.ui.background__light.hex};

  display: flex;
  align-items: center;
```

Then the properties of the box itself are set first, starting with the height, and then a new block with the properties governing its children.